src/kernel-install/test-kernel-install.sh

```diff
@@ -269,3 +269,47 @@ test -d "$BOOT_ROOT/hoge/1.1.1"
 test ! -e "$BOOT_ROOT/hoge/1.1.1"
 test -d "$BOOT_ROOT/hoge"
 rmdir "$BOOT_ROOT/hoge"
+
+###########################################
+# tests for --json=
+###########################################
+output="$("$kernel_install" -v --json=pretty inspect 1.1.1 "$D/sources/linux")"
+
+diff -u <(echo "$output") - <<EOF
+{
+	"MachineID" : "3e0484f3-634a-418b-8e6a-39e8828b03e3",
+	"KernelImageType" : "unknown",
+	"Layout" : "other",
+	"\$BOOT" : "$BOOT_ROOT",
```

Comment:
we shouldn't do this, i.e. field names with weird chars. Please rename the JSON field to something more compatible with other programming languages.

BootLocation? BootPath? someting like that.